packages/management/addons-manager/bundle/config/upstream/addons-manager.yaml

```diff
@@ -246,7 +246,7 @@ spec:
       - args:
         - --metrics-bind-addr=0
         - --leader-elect=false
-        - --health-addr=:18316
+        - #@ "--health-addr=:{}".format(data.values.tanzuAddonsManager.deployment.healthzPort)
         - #@ "--addon-namespace={}".format(data.values.tanzuAddonsManager.namespace)
         #@ if/end data.values.tanzuAddonsManager.featureGates.clusterBootstrapController:
         - --feature-gate-cluster-bootstrap=true
```

Comment:
Where is the param `webhook-server-port` being passed?

Reply:
@vijaykatam which one should we use?
`webhook-server-port=:9865` or `webhook-server-port=9865`

Reply:
`- #@ "--webhook-server-port=:{}".format(data.values.tanzuAddonsManager.deployment.webhookServerPort)`

Reply:
added webhook-server-port in the args